Source/WebCore/loader/ImageLoader.h

```diff
@@ -147,6 +152,7 @@ class ImageLoader : public CachedImageClient {
     bool m_imageComplete : 1;
     bool m_loadManually : 1;
     bool m_elementIsProtected : 1;
+    bool m_microtaskQueued : 1;
```

Comment:
I would suggest renaming this flag `m_isMicrotaskQueued` and the setter and getter to be 
```
bool isMicrotaskQueued() const
void setIsMicrotaskQueued(bool isMicrotaskQueued)
```